tests/test_models.py

```diff
@@ -151,6 +151,17 @@ def test_model1(dmx_psrs,caplog):
     m1=models.model_1(dmx_psrs,noisedict=noise_dict)
     assert hasattr(m1,'get_lnlikelihood')
 
+def test_modelramp(nodmx_psrs, caplog):
+    tmax = max([p.toas.max() for p in nodmx_psrs])
+    tmin = min([p.toas.min() for p in nodmx_psrs])
+    Tspan = tmax - tmin
+
+    m=models.model_ramp(nodmx_psrs) # should I be testing the Log and Lookup Likelihoods?
```

Comment:
Tests in `e_e` should test the actual `e_e` functions, in this case `model_bwm` and `model_ramp`, which in turn call `blocks`.  If the new lookup likelihood is part of `enterprise`, it should be tested there.

Reply:
Is this test ok? I'll take a look at the tests for blocks and add those as well.

Reply:
Also, is it ok to add things like manual specifications to the amplitudes in blocks? IIRC, those parameters weren't originally there.